main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun: amplifier
+//  action 1: guitar is connected to the amplifier
+amplifier.guitarConnected();

```

Comment:
choose actions that you can express as present-tense plain-English statements.
avoid `is` in the plain-English action.

Also, you need to choose a plain-english expression that involves the amplifier DOING something.
the amplifier isn't doing anything here. 
The guitar is the one doing something. 
- [ ] revise